hudi-utilities/src/main/java/org/apache/hudi/utilities/streamer/HoodieStreamer.java

```diff
@@ -747,6 +747,7 @@ public StreamSyncService(Config cfg, HoodieSparkEngineContext hoodieSparkContext
             cfg.baseFileFormat = "PARQUET"; // default for backward compatibility
           }
           Map<String, String> propsToValidate = new HashMap<>();
+          properties.get().setProperty(HoodieTableConfig.PAYLOAD_CLASS_NAME.key(), cfg.payloadClassName);
```

Comment:
can we also add other merge properties to this list. 
`hoodie.write.record.merge.mode` 
and `hoodie.write.record.merge.strategy.id`

Reply:
in case of HoodieStreamer, we have top level configs only for these 
```
--merge-mode
--merge-strategy-id 
```